src/python/py/models/builder.py

```diff
@@ -1709,6 +1894,8 @@ def create_model(model_name, input_path, output_dir, precision, execution_provid
             onnx_model = MistralModel(config, io_dtype, precision, execution_provider, cache_dir, extra_options)
         elif config.architectures[0] == "PhiForCausalLM":
             onnx_model = PhiModel(config, io_dtype, precision, execution_provider, cache_dir, extra_options)
+        elif config.architectures[0] == "MixtralForCausalLM":
```

Comment:
Can you add the new architecture such that the alphabetical order is maintained? This helps quickly identify which architectures are currently supported.